sql/mob_pools.sql

```diff
@@ -5953,131 +5953,131 @@ INSERT INTO `mob_pools` VALUES (5892,'Euryale','Euryale',171,0x06007106000000000
+INSERT INTO `mob_pools` VALUES (5979,'selh_teus','Selh\'teus',477,0x0000160C00000000000000000000000000000000,7,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5980,'yoran-oran_uc','Yoran-Oran',153,0x0000170C00000000000000000000000000000000,3,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5981,'sylvie_uc','Sylvie',149,0x0000180C00000000000000000000000000000000,21,3,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5982,'abquhbah','Abquhbah',149,0x00001A0C00000000000000000000000000000000,1,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5983,'balamor','Balamor',492,0x00001B0C00000000000000000000000000000000,8,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5984,'august','August',149,0x00001C0C00000000000000000000000000000000,7,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5985,'rosulatia','Rosulatia',455,0x00001D0C00000000000000000000000000000000,4,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5986,'teodor','Teodor',149,0x00001F0C00000000000000000000000000000000,4,6,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5987,'ullegore','Ullegore',74,0x0000210C00000000000000000000000000000000,4,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5988,'makki-chebukki','Makki-Chebukki',153,0x0000220C00000000000000000000000000000000,11,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5989,'king_of_hearts','KingOfHearts',61,0x0000230C00000000000000000000000000000000,5,3,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5990,'morimar','Morimar',149,0x0000240C00000000000000000000000000000000,9,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5991,'darrcuiln','Darrcuiln',489,0x0000250C00000000000000000000000000000000,1,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5992,'aahm','ArkHM',149,0x0000290C00000000000000000000000000000000,13,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5993,'aaev','ArkEV',145,0x00002A0C00000000000000000000000000000000,7,3,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5994,'aamr','ArkMR',151,0x00002B0C00000000000000000000000000000000,9,6,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5995,'aatt','ArkTT',153,0x00002C0C00000000000000000000000000000000,4,8,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5996,'aagk','ArkGK',146,0x00002D0C00000000000000000000000000000000,12,14,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5997,'iroha','Iroha',149,0x0000270C00000000000000000000000000000000,4,12,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
```

Comment:
Wiki is similarly confused for Iroha, listing her as SAM on the main page, but weirdly BLM/SAM on her specific page. It also lists her ROV version as WHM/SAM on its individual page, instead of SAM/WHM as it does on the main category listing.

Absent some concrete argument for main mage, I'd very much bet that Iroha is main SAM. Just a feeling 😉